portfolio/src/main/webapp/script.js

```diff
@@ -55,21 +55,49 @@ function pauseProjectChange() {
   clearTimeout(changeCurrProj);
 }
 
-/** Fetches comment(s) and updates the UI to display */
-function fetchCommentAndDisplay() {
+/** Fetches comment(s) and updates the UI to display them */
+function fetchCommentAndDisplay(numOfCommentsToDisplay) {
   fetch('/data').then(response => response.json()).then((comments) => {
-    const commentsContainer = document.getElementById('comments-container');
-    comments.forEach((comment) => {
-      const currentCommentContainer = createDivElement();
-      currentCommentContainer.style.padding = "50px 0px";
-      commentsContainer.appendChild(currentCommentContainer);
-      currentCommentContainer.appendChild(createParagraphElement(comment.name));
-      currentCommentContainer.appendChild(createParagraphElement(comment.email));
-      currentCommentContainer.appendChild(createParagraphElement(comment.text));
-    });
+    /** 
+    * Error check to ensure the user did not select to display more comments than exist in the datastore
+    *
+    * UI is not updated on user selecting to view more comments than exist in the datastore
+    */ 
+    if (numOfCommentsToDisplay > comments.length) {
+      alert("The selected value is greater than the total number of comments.");
+    }
```

Comment:
As a user I don't know how many comments are in the database. If I go over, shouldn't I still be able to see all the comments?